### ISSUE TITLE
set_valid_wd fails if file instead of directory given

### ISSUE THREAD

**Issue by barryrowlingson** [christophergandrud/simpleSetup#1]
Error if asked to set to an existing file rather than a directory:

```
> set_valid_wd(c("/etc","/"))
Working directory set to: /etc
> set_valid_wd(c("/etc/passwd","/"))
Error in setwd(i) : cannot change working directory
```

Maybe it should use `dir.exists` instead of `file.exists`? Or warn if its given a file when it should expect only directories?

**Comment by christophergandrud** [comment]
good idea.

**Comment by christophergandrud** [comment]
Implemented in 759fc8d9bd0642bff9bd5df8cfef9f04e4bf413a
